src/Mod/PartDesign/Gui/TaskLinearPatternParameters.cpp

```diff
@@ -57,104 +56,22 @@ TaskLinearPatternParameters::TaskLinearPatternParameters(ViewProviderTransformed
     : TaskTransformedParameters(TransformedView, parent)
     , ui(new Ui_TaskLinearPatternParameters)
 {
-    // we need a separate container widget to add all controls to
-    proxy = new QWidget(this);
-    ui->setupUi(proxy);
-    QMetaObject::connectSlotsByName(this);
-
-    this->groupLayout()->addWidget(proxy);
-
-    ui->buttonOK->hide();
-    ui->checkBoxUpdateView->setEnabled(true);
-
-    selectionMode = none;
-
-    blockUpdate = false; // Hack, sometimes it is NOT false although set to false in Transformed::Transformed()!!
     setupUI();

```

Comment:
In this Ctor you are removing previous stuff but you are not adding your new `setupParameterUI()` is that not breaking something?

Reply:
The ``setupUi()`` function comes from ``TaskTransformedParameters``. It will setup the common ui and then call the ``setupParameterUI()`` function. This is used in the ctors that create the standalone features. The ctors for the MultiTransform subfeatures don't need the common ui and call ``setupParameterUI()`` directly.
https://github.com/FreeCAD/FreeCAD/blob/24c138373c28674d90f18507bb95ebd91b07f048/src/Mod/PartDesign/Gui/TaskTransformedParameters.cpp#L93-L135

Some of the old code used a private ``setupUi()`` function, I resused the name.